.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: node_js
 node_js:
-  - "0.8"
+  - "0.12"
```

Comment:
Sure, I just added the other versions to the travis.yml file.

There is one piece of the travis test that's failing, though. The `./integration-tests.sh` file is returning a failing code although there aren't any actual errors being returned.